src/blog/lagom-1-3-7.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Lagom 1.3.7 is released!
+date: 2017-08-18
+author_github: rcavalcanti
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.3.7
+---
+
+The Lagom team has released Lagom 1.3.7. For a list of the specific changes in Lagom 1.3.7, see the [change log](/changelog.html).
+
+As an overview, this release contains the following:
+
+- Update Akka to 2.4.20 - (Security Patch Release)[http://akka.io/blog/news/2017/08/10/akka-2.4.20-released] 
+- Changed logging default settings - file appender removed (see details below)
+- Experimental Service Gateway based on akka-http
```

Comment:
> About not supporting netty anymore, I think we still need to as long as the akka-http service gateway is considered experimental.

What I mean by that is that we don't plan to fix any bugs in the netty implementation that are already fixed in the akka-http one. There are already a couple known issues in this category (some problem with multipart uploads, and one with URL encoding of slashes).

I think we want to make the switch pretty quickly... in 1.4 akka-http will be the default and netty will only be there in case of regressions. In 1.5 I think we should drop the netty implementation, if there are no known issues with the akka-http one.